kustomize/components/shopping-assistant/scripts/1_deploy_alloydb_infra.sh

```diff
@@ -82,7 +82,8 @@ gcloud alloydb instances create ${ALLOYDB_INSTANCE_NAME}-replica \
 gcloud beta alloydb instances update ${ALLOYDB_INSTANCE_NAME} \
     --cluster=${ALLOYDB_CLUSTER_NAME} \
     --region=${REGION} \
-    --assign-inbound-public-ip=ASSIGN_IPV4
+    --assign-inbound-public-ip=ASSIGN_IPV4 \
+    --database-flags password.enforce_complexity=on
```

Comment:
This flag is now required or else the AlloyDB instance update fails with an error.